app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
WRT to **jshint** limitations ... makes sense; have been away from jshint for a bit and had thought maybe it supported ES6 more fully now.